route/build/src/ascii_utils.f90

```diff
@@ -204,8 +204,10 @@ SUBROUTINE get_vlines(unt,vlines,err,message)
   vlines(current%ix) = current%chardat
   previous=>current; current=>current%next
   deallocate(previous)
+  nullify(previous)
  end do
- if(associated(list)) nullify(list)
+ nullify(list)
+ !if(associated(list)) nullify(list)
```

Comment:
Just remove the commented out line.

Reply:
Yes,  I think this can be safely removed

Reply:
OK